src/app/cluster/cluster-details/version-picker/version-picker.component.ts

```diff
@@ -52,22 +52,24 @@ export class VersionPickerComponent implements OnInit, OnChanges {
     this.downgradesAvailable = false;
     this.someUpgradesRestrictedByKubeletVersion = false;
 
-    this.upgrades.forEach(upgrade => {
-      const isUpgrade = lt(this.cluster.spec.version, upgrade.version);
-      const isDowngrade = gt(this.cluster.spec.version, upgrade.version);
+    if (this.upgrades) {
```

Comment:
This shouldn't be needed. It is always initialized.

Reply:
> It is always initialized.

It isn't. I produces an error during the cluster creation. I can revert this as this is not connected to my change. I just wanted to keep the console clean.

Reply:
I'll revert since this is probably an issue on the API side.